pkg/plugin/test-keeper/plugin/comment_message_test.go

```diff
@@ -189,6 +189,24 @@ var _ = Describe("Test keeper comment message creation", func() {
 				"https://raw.githubusercontent.com/owner/repo/46cb8fac44709e4ccaae97448c65e8f7320cfea7/" +
 					"http/server.com/custom_message_file.md"))
 		})
+
+		It("should create message taken from a string set in config", func() {
+			// given
+			url := "http://github.com/my/repo/test-keeper.yaml"
+			config := plugin.TestKeeperConfiguration{
+				PluginConfiguration: config.PluginConfiguration{
+					LocationURL: url,
+					PluginHint:  "Custom message",
+				},
+			}
+
+			// when
+			msg := plugin.CreateCommentMessage(config, scm.RepositoryChange{})
+			sanitizedMsg := removeHtmlElements(msg)
```

Comment:
If it's a custom message should we have an icon added regardless? WDYT @MatousJobanek?

Reply:
I would remove it. I can imagine that somebody can see the icon as too big so having the custom hint is also a way to get rid of the icon and make the comment smaller. The only thing that should stay are the titles, the rest is should be replaced by the custom message (that can contain some icons as well)